### DISCUSSION TITLE
expended is returned List!!

### DISCUSSION THREAD

**Discussion by jhh78** [pocketbase/pocketbase#4368] [Q&A] [answered]
greetings.

My current table structure is like this.
![image](https://github.com/pocketbase/pocketbase/assets/51219166/93b32a6f-6f6d-4c00-9b9e-37af0f2eaf4b)

and DM collection info is
<img width="679" alt="image" src="https://github.com/pocketbase/pocketbase/assets/51219166/26b2a5b2-8545-49d6-bdc6-daaa23183ab4">

I ran the code.
And what I wanted was object type user information.
However, list-type user information is being returned.
![image](https://github.com/pocketbase/pocketbase/assets/51219166/c96db0ca-0e8d-48c5-a382-c61c1688e0e0)
![image](https://github.com/pocketbase/pocketbase/assets/51219166/32877adf-f878-471b-82af-865d020d2c6e)

If there are any issues that I don't know about, please let me know.

**Comment by ganigeorgiev** [comment, marked as answer]
Are you testing with the Dart SDK?

If yes, then the `RecordModel.expand` data value will be always `List<RecordModel>` for simplicity and to avoid using `dynamic` (`RecordModel.toJson()` should output the record data as it was originally loaded).

**Reply by jhh78** [reply]
Yes, using dart.
The question has been resolved.
